src/Artsy/Router/v2/makeClientApp.tsx

```diff
@@ -0,0 +1,45 @@
+import { createRelaySSREnvironment } from "Artsy/Relay/createRelaySSREnvironment"
+import React from "react"
+import { getUser } from "Utils/user"
+import { Boot } from "../Boot"
+import { createRouteConfig } from "../Utils/createRouteConfig"
+import { buildClientApp } from "./buildClientApp2"
+
+export function makeClientApp(config) {
```

Comment:
The `buildClientApp` / `makeClientApp` language is a kinda meh, but when the router is moved out of reaction `buildClient/ServerApp` naming will change. 

`makeClientApp` could be more descriptive tho. `clientRouteAdaptor`? `clientApp`? Can likely worry about this later as a follow-up item.